monkey/monkey_island/cc/ui/webpack.config.js

```diff
@@ -1,33 +1,56 @@
 const path = require('path');
-const HtmlWebPackPlugin = require("html-webpack-plugin");
 
-module.exports = {
+var isProduction = process.argv[process.argv.indexOf('--mode') + 1] === 'production';
```

Comment:
Did you open a PR on labs to call it with production argument if it's a release build?